starter-code/src/main.js

```diff
@@ -1,50 +1,143 @@
-  { name: 'spiderman',       img: 'spiderman.jpg' },
-  { name: 'superman',        img: 'superman.jpg' },
-  { name: 'the avengers',    img: 'the-avengers.jpg' },
-  { name: 'thor',            img: 'thor.jpg' }
+var cards = [{
+    name: 'aquaman',
+    img: 'aquaman.jpg'
+  },
+  {
+    name: 'batman',
+    img: 'batman.jpg'
+  },
+  {
+    name: 'captain america',
+    img: 'captain-america.jpg'
+  },
+  {
+    name: 'fantastic four',
+    img: 'fantastic-four.jpg'
+  },
+  {
+    name: 'flash',
+    img: 'flash.jpg'
+  },
+  {
+    name: 'green arrow',
+    img: 'green-arrow.jpg'
+  },
+  {
+    name: 'green lantern',
+    img: 'green-lantern.jpg'
+  },
+  {
+    name: 'ironman',
+    img: 'ironman.jpg'
+  },
+  {
+    name: 'spiderman',
+    img: 'spiderman.jpg'
+  },
+  {
+    name: 'superman',
+    img: 'superman.jpg'
+  },
+  {
+    name: 'the avengers',
+    img: 'the-avengers.jpg'
+  },
+  {
+    name: 'thor',
+    img: 'thor.jpg'
+  },
+  {
+    name: 'aquaman',
+    img: 'aquaman.jpg'
+  },
+  {
+    name: 'batman',
+    img: 'batman.jpg'
+  },
+  {
+    name: 'captain america',
+    img: 'captain-america.jpg'
+  },
+  {
+    name: 'fantastic four',
+    img: 'fantastic-four.jpg'
+  },
+  {
+    name: 'flash',
+    img: 'flash.jpg'
+  },
+  {
+    name: 'green arrow',
+    img: 'green-arrow.jpg'
+  },
+  {
+    name: 'green lantern',
+    img: 'green-lantern.jpg'
+  },
+  {
+    name: 'ironman',
+    img: 'ironman.jpg'
+  },
+  {
+    name: 'spiderman',
+    img: 'spiderman.jpg'
+  },
+  {
+    name: 'superman',
+    img: 'superman.jpg'
+  },
+  {
+    name: 'the avengers',
+    img: 'the-avengers.jpg'
+  },
+  {
+    name: 'thor',
+    img: 'thor.jpg'
+  }
 ];
 
-$(document).ready(function(){
+$(document).ready(function () {
   var memoryGame = new MemoryGame(cards);
   var html = '';
   memoryGame.cards.forEach(function (pic, index) {
     html += '<div class= "card" id="card_' + pic.name + '">';
     html += '<div class="back"';
-    html += '    name="'       + pic.img +  '">';
+    html += '    name="' + pic.img + '">';
     html += '</div>';
     html += '<div class="front" ';
-    html += 'style="background: url(img/' + pic.img + '") no-repeat"';
+    html += 'style="background: url(img/' + pic.img + ') no-repeat">';
     html += '</div>';
     html += '</div>';
   });
 
+  console.log(html);
+
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
-});
+  var firstCard = null;
+  $('.back').on('click', function () {
+
+    $(this).toggle();
+    $(this).siblings().toggle();
+    var cardImage = $(this).attr('name');
+    for (let i = 0; i < cards.length; i++) {
+      if (cardImage === cards[i].img) {
+        if (firstCard === null) {
+          firstCard = cards[i];
+        } else {
+          var result = memoryGame.checkIfPairs (firstCard, cards[i]);
```

Comment:
This is bugging because the method is named `checkIfPair`, so it's not finding the method `checkIfPairs` :)